### PR TITLE
Excessive requests to container lookup

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #7788 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/notifications/BannerNotifications.js [hunk 1]
```diff
@@ -115,7 +119,9 @@ export default function BannerNotifications() {
 			) }
 			{ ga4ModuleConnected && hasGTMScope && isGA4ModuleOwner && (
 				<Fragment>
-					<GoogleTagIDMismatchNotification />
+					{ !! hasMismatchedGoogleTagID && (
```
Comment (kuasha420) [inline assets/js/components/notifications/BannerNotifications.js:122]: I don't think we need to convert this to boolean, as the value itself is always a boolean.

**Review by kuasha420 — CHANGES_REQUESTED** [review]
Thank you @zutigrm for the PR. Code works well in my testing in both Authenticated and shared dashboard. :heavy_check_mark: 

Just one little nit.

File: assets/js/components/notifications/BannerNotifications.js (hunk 1 above)
Comment (zutigrm) [reply assets/js/components/notifications/BannerNotifications.js]: @kuasha420 You are correct, I kinda thought to double ensure it. But it doesn't seem needed. Updated

**Review by kuasha420 — APPROVED** [review]
Thanks @zutigrm. This is now good to go! :rocket:

**Pull request merged**